Source/WebKit/UIProcess/LegacyGlobalSettings.h

```diff
@@ -52,6 +52,9 @@ class LegacyGlobalSettings {
     const HashSet<String>& schemesToRegisterAsNoAccess() { return m_schemesToRegisterAsNoAccess; }
     void registerURLSchemeAsNoAccess(const String& scheme) { m_schemesToRegisterAsNoAccess.add(scheme); }
 
+    const HashSet<String>& hostnamesToRegisterAsLocal() { return m_hostnamesToRegisterAsLocal; }
```

Comment:
This getter should be marked as const:
`const HashSet<String>& hostnamesToRegisterAsLocal() const { return m_hostnamesToRegisterAsLocal; }`